src/Workspaces/Remote/ServiceHub/UserOperationBooster.cs

```diff
@@ -31,10 +31,18 @@ public static UserOperationBooster Boost()
             {
                 s_count++;
 
-                if (s_count == 1)
+                try
                 {
-                    // boost to normal priority
-                    Process.GetCurrentProcess().PriorityClass = ProcessPriorityClass.Normal;
+                    if (s_count == 1)
+                    {
+                        // boost to normal priority
+                        Process.GetCurrentProcess().PriorityClass = ProcessPriorityClass.Normal;
+                    }
+                }
+                catch (Exception)
```

Comment:
```suggestion
                catch (Exception e) when (e is PlatformNotSupportedException || e is Win32Exception)
```

Reply:
@Youssef1313 would this be better?

Reply:
I'm not sure if Roslyn team prefers it that way or they prefer multiple catch statements.
You can wait for an official review on this one.